### DISCUSSION TITLE
Suggestions on starting a new community!

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1764] [General] [answered]
Hello everyone, This is just a random discussion. I just wanted to know that if anyone had to start a new community (Both on discord and GitHub) what are the essential things that are needed to be kept in mind in order to create a good supportive community (one like EddieHub) and how shall we proceed to scale it further?

**Comment by ghost** [comment]
- [ ] Use something like EddieBot To Ensure Inclusive Language
 - [ ] Add a Support repository
 - [ ] Have a YT Channel
 - [ ] GitHub - Open To Join
 - [ ] Add some cool repositories like awesome-github-profiles and all
 All of what I did in @FriendHubComm and What I am going to do in my community

**Comment by kaiwalyakoparkar** [comment, marked as answer]
Adding some points
- Spead awareness of your community
- Take session and try to gather like minded folks
- Take feedback of problems they are facing
- Facilitate them with proper session to enable them grow

**Reply by ghost** [reply]
> Take session and try to gather like minded folks

Although I am not planning to create a community, if I create one then then first person, who will I try add on my community is you and kunal, then nhcarrigan, Vyvy-vi and Karuna

**Reply by kaiwalyakoparkar** [reply]
Yeah! thanks! I am planning to start one this month. If you are interested let me know. Let's see how it goes!

**Reply by ghost** [reply]
Ya I am interested once you create, Invite me as well! Don't forget to partner it with EddieHub!

**Reply by kaiwalyakoparkar** [reply]
> Ya I am interested once you create, Invite me as well! Don't forget to partner it with EddieHub!

That is for sure!
